### PR TITLE
SIG-2165 Add eIDAS Login verification

### PR DESCRIPTION
**Pull request by ATimmeh33**
Adds a verification object for the eIDAS Login

### CODE REVIEW

File: src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		/// </summary>
+		public string FirstName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the LastName.
+		/// </summary>
+		public string LastName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the DateOfBirth.
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs]: Gets or sets the date of birth

File: src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		/// </summary>
+		public string Level { get; set; }
+
+		/// <summary>
+		/// Gets or sets the FirstName.
+		/// </summary>
+		public string FirstName { get; set; }
+
+		/// <summary>
+		/// Gets or sets the LastName.
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs]: Gets or sets the last name

File: src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		/// </summary>
+		public string Uid { get; set; }
+
+		/// <summary>
+		/// Gets or sets the Level.
+		/// </summary>
+		public string Level { get; set; }
+
+		/// <summary>
+		/// Gets or sets the FirstName.
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs]: Gets or sets the authenicated users first name

File: src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+		/// <summary>
+		/// Gets or sets the Uid.
+		/// </summary>
+		public string Uid { get; set; }
+
+		/// <summary>
+		/// Gets or sets the Level.
+		/// </summary>
+		public string Level { get; set; }
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs]: Perhaps we should enum this.

File: src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+	public class EidasLoginVerification
+		: IVerification
+	{
+		/// <summary>
+		/// Gets the <see cref="IVerification.Type"/>.
+		/// </summary>
+		public string Type { get; } = "eIDAS Login";
+
+		/// <summary>
+		/// Gets or sets the Uid.
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/DataObjects/EidasLoginVerification.cs]: Gets or sets the uid.

File: src/SignhostAPIClient/Rest/JsonConverters/LevelEnumConverter.cs [hunk 6]
```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Reflection;
+using Newtonsoft.Json;
+using Signhost.APIClient.Rest.DataObjects;
+
+namespace Signhost.APIClient.Rest.JsonConverters
+{
+	/// <inheritdoc/>
```
Comment (MrJoe) [inline src/SignhostAPIClient/Rest/JsonConverters/LevelEnumConverter.cs]: I am fine with the inheritdoc on all the inheritted properties. But some summary on what this converter is for might be welcome.

**Comment by ATimmeh33** [comment]
Can be merged after eIDAS production release

**Pull request merged**